site.mk

```diff
@@ -1,12 +1,11 @@
 GLUON_FEATURES := \
 	autoupdater \
+	config-mode-outdoor \
 	ebtables-filter-multicast \
 	ebtables-filter-ra-dhcp \
-	ebtables-limit-arp \
```

Comment:
If feasible I think we should keep limit-arp, at least as long as we use Singledomain on a large scale.